packages/editor/src/store/actions.js

```diff
@@ -44,15 +46,20 @@ export function resetPost( post ) {
  * Returns an action object used in signalling that the latest autosave of the
  * post has been received, by initialization or autosave.
  *
- * @param {Object} post Autosave post object.
+ * @param {Object} newAutosave Autosave post object.
  *
  * @return {Object} Action object.
  */
-export function resetAutosave( post ) {
-	return {
-		type: 'RESET_AUTOSAVE',
-		post,
-	};
+export function resetAutosave( newAutosave ) {
+	deprecated( 'resetAutosave action (`core/editor` store)', {
+		alternative: 'receiveAutosave action (`core` store)',
+		plugin: 'Gutenberg',
+	} );
+
+	const postId = select( 'core' ).getCurrentPostId();
```

Comment:
`'core'` doesn't have `getCurrentPostId`. I assume it should be `'core/editor'`.

Reply:
Yep, thanks for catching that mistake.